### PR TITLE
Update mapmerge2

### PR DESCRIPTION
**Pull request by PsiOmegaDelta**
Updating didn't help with the newline issue but otherwise appears to work.

Main addition is `update_paths.py` which is a way to automatically replace paths, etc. See the script itself as well as examples over at https://github.com/tgstation/tgstation/tree/master/tools/mapmerge2/map_scripts

### CODE REVIEW

File: tools/hooks/install.sh [hunk 1]
```diff
@@ -9,4 +10,11 @@ for f in *.merge; do
 	echo Installing merge driver: ${f%.merge}
 	git config --replace-all merge.${f%.merge}.driver "tools/hooks/$f %P %O %A %B %L"
 done
+
+echo "Installing tgui hooks"
```
Comment (comma) [inline tools/hooks/install.sh]: we dun have TGUI

Comment (PsiOmegaDelta) [reply tools/hooks/install.sh]: This is indeed true

**Review by comma — APPROVED** [review]

File: tools/mapmerge2/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-pygit2>=0.27.2
-bidict>=0.13.1
-Pillow>=5.1.0
+pygit2==1.0.1
+bidict==0.13.1
+Pillow==7.0.0
```
Comment (afterthought2) [inline tools/mapmerge2/requirements.txt:3]: What happens if you have higher versions installed?

Comment (PsiOmegaDelta) [reply tools/mapmerge2/requirements.txt:3]: Good question apparently without a direct answer, just based on a quick search. I'll get back to you

Comment (deathride58) [reply tools/mapmerge2/requirements.txt:3]: The py script installs and uses the specific version listed in requirements.txt, and pip is responsible for keeping the specific version separate from whatever you've got installed.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**